CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## v0.10.4dev
 [Full Changelog](https://github.com/SeldonIO/alibi-detect/compare/v0.10.3...master)
 
+### Added
+- **New feature** MMD drift detector has been extended with a [KeOps](https://www.kernel-operations.io/keops/index.html) backend to scale and speed up the detector. 
+See the [documentation](https://docs.seldon.io/projects/alibi-detect/en/latest/cd/methods/mmddrift.html) and [example notebook](https://docs.seldon.io/projects/alibi-detect/en/latest/examples/cd_mmd_keops.html) for more info ([#548](https://github.com/SeldonIO/alibi-detect/pull/548)).
```

Comment:
Would you be able to change the `latest` links to `stable` please? (see https://github.com/SeldonIO/alibi-detect/pull/479).

Reply:
Ok, my reasoning was just that you would want to check what has changed since the last release from the changelog before the actual next release.

Reply:
Discussed offline. Makes sense to keep links under the dev version as `latest`, and change to `stable` upon release. Otherwise, the `stable` links in dev might not be valid (i.e. if they are links to new pages)...